kratos/processes/tetrahedral_mesh_orientation_check.cpp

```diff
@@ -51,7 +51,8 @@ void TetrahedralMeshOrientationCheck::Execute()
         GeometryType& r_geometry = it_elem->GetGeometry();
         const GeometryData::KratosGeometryType geometry_type = r_geometry.GetGeometryType();
 
-        if (geometry_type == GeometryData::KratosGeometryType::Kratos_Tetrahedra3D4 || geometry_type == GeometryData::KratosGeometryType::Kratos_Triangle2D3) {
+        if (geometry_type == GeometryData::KratosGeometryType::Kratos_Tetrahedra3D4 || geometry_type == GeometryData::KratosGeometryType::Kratos_Triangle2D3 || //acceptable linear geoms
```

Comment:
I would suggest to have a IsAcceptable() method instead of copy pasting the code several times